src/builtins/Transformers.jl

```diff
@@ -900,10 +900,19 @@ end
 
 # If v=categorical('a', 'a', 'b', 'a', 'c') and MMI.int(v[1]) = ref
 # then `_hot(v, ref) = [true, true, false, true, false]`
-_hot(v::AbstractVector{<:CategoricalValue}, ref) = map(v) do c
+hot(v::AbstractVector{<:CategoricalValue}, ref) = map(v) do c
     MMI.int(c) == ref
 end
 
+function hot(col::AbstractVector{<:Union{Missing, CategoricalValue}}, ref) map(col) do c  
```

Comment:
Very minor style point: You have some extraneous whitespace at the end of line 907 and elsewhere. Your editor probably has a way to highlight superfluous whitespace and delete it. (In emacs, which I use, there is whitespace-mode).

Reply:
Sure. I changed the settings now